win32/src/winapi/kernel32/dll.rs

```diff
@@ -322,8 +322,11 @@ unsafe impl ::memory::Pod for STARTUPINFOA {}
 pub fn GetStartupInfoA(_machine: &mut Machine, lpStartupInfo: Option<&mut STARTUPINFOA>) -> u32 {
     // MSVC runtime library passes in uninitialized memory for lpStartupInfo, so don't trust info.cb.
     let info = lpStartupInfo.unwrap();
-    let len = std::cmp::min(info.cb, std::mem::size_of::<STARTUPINFOA>() as u32);
+    let len = std::mem::size_of::<STARTUPINFOA>() as u32;
```

Comment:
By the way, I checked and I now believe that STARTUPINFOA has never changed in size, so my worries about callers passing in an undersized buffer or something like that are unfounded.  And further I checked and at least win2k's kernel32.dll does exactly what you've done here, so I think this is correct.